### PR TITLE
Updated Unit test cases  and Readme.md for Pathways

### PR DESCRIPTION
**Pull request by sujata-m**
This PR contains - 

1. Updated all unit test cases
2. Unit test cases no longer require .env file
3. Fixed test harness issue.
4. Updated test folder structure.
5. Updated Readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,19 @@ Add the new set of test inside `test/tests.json` file like -
 
 ```
 
+#### How to run the tests
+1. Set up the python env locally, see [here](#how-to-setup-aand-build) 
+2. Set up `.env` file, see [here](#connectivity-to-cloud) 
+3. To run the tests
+    ```
+   # To run all th unit test cases
```
Comment (susrisha) [inline README.md]: To run all _the_

File: README.md [hunk 2]
```diff
@@ -77,6 +77,19 @@ Add the new set of test inside `test/tests.json` file like -
 
 ```
 
+#### How to run the tests
+1. Set up the python env locally, see [here](#how-to-setup-aand-build)
```
Comment (susrisha) [inline README.md]: two `a` characters for `and`

**Review by susrisha — CHANGES_REQUESTED** [review]
one minor typo

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
I see that 'fail_riles_1.zip' and 'success_1_all_attr.zip' are under src/assets folder. As far as I know these two are zip files for tests. Shouldn't they be under tests folder? They unnecessarily take up space in the docker container.

**Review by susrisha — APPROVED** [review]

**Review by sureshgaussian — APPROVED** [review]
Tested the code locally and it passes

**Pull request merged**